README.md

```diff
@@ -201,11 +201,23 @@ kind: RunnerDeployment
 metadata:
   name: summerwind-actions-runner-controller
 spec:
-  minReplicas: 1
-  maxReplicas: 3
   template:
     spec:
       repository: summerwind/actions-runner-controller
+---
+apiVersion: actions.summerwind.dev/v1alpha1
+kind: HorizontalRunnerAutoscaler
+metadata:
+  name: summerwind-actions-runner-controller
```

Comment:
This name is confusing, so I think it would be better to use a name like `example-runner-deployment-autoscaler`